src/bin/probot.ts

```diff
@@ -13,7 +13,7 @@ dotenvConfig();
 const __dirname = dirname(fileURLToPath(import.meta.url));
 const pkg = loadPackageJson(resolve(__dirname, "package.json"));
 
-if (!isSupportedNodeVersion()) {
+if (!isSupportedRuntime(globalThis)) {
```

Comment:
This would need to be adapted to only log the message if it's an unsupported Node version, and not the other platforms

Reply:
I know, but as currently only node is officially supported, I did not want to go into the deep regarding the log message. E.g. we could consider to output a message based on the runtime.

Reply:
can we postpone the message discussion?